packages/stores/types.ts

```diff
@@ -603,17 +604,10 @@ type TContractTradeStore = {
     last_contract: TContractStore | Record<string, never>;
     markers_array: Array<{
         type: string;
-        contract_info: {
-            accu_barriers_difference:
-                | boolean
-                | {
-                      top: string;
-                      bottom: string;
-                      font: string;
-                  };
-            has_crossed_accu_barriers: boolean;
-            is_accumulator_trade_without_contract: boolean;
-        };
+        contract_info: ProposalOpenContract &
```

Comment:
I tried to import `TContractInfo` from `@shared`, but got an error that in can't be used inside of Array. Weird, but the same code without type name worked.

Reply:
@kate-deriv :) 
```suggestion
        contract_info: TPortfolioPosition['contract_info'];
```

Reply:
Will do, thank you